src/Commands/Developer/birthday.js

```diff
@@ -0,0 +1,35 @@
+const moment = require("moment");
+
+module.exports = {
+  name: 'birthday',
+  aliases: [],
+  category: 'Developer',
+  description: 'I spit out an error.',
+  usage: '',
+  permissions: 'SEND_MESSAGES',
+  clientPerms: 'SEND_MESSAGES',
+  creatorOnly: true,
+  guildOnly: false,
+  premiumOnly: false,
+  requiresArgs: true,
+  run: async (client, msg, args) => {
+
+    const day = args.join(" ")
+
+    const birthdate = moment(day).format("DD/MM/YYYY");
+
+    if(!birthdate) return msg.channel.send("Incorrect birthdate")
+
+    var { UserProfile } = client.Models
+
+    var query = { userID: msg.author.id }
+
+    const data = await UserProfile.find({ userID: msg.author.id })
```

Comment:
data isn't used?

Reply:
you can remove it, i just added it for other things to make sure it saved like data.save() but i voided that